src/dotnet/CarbonAware/Model/EmissionsData.cs

```diff
@@ -7,7 +7,6 @@ public record EmissionsData
     public DateTime Time { get; set; }
     public double Rating { get; set; }
 
-
     public bool TimeBetween(DateTime fromNotInclusive, DateTime? endInclusive)
```

Comment:
Should this be in EmissionsData or just a general utility function? Not sure if this logic is needed for other DateTime values